prov/gni/include/fi_ext_gni.h

```diff
@@ -62,6 +62,7 @@ typedef enum dom_ops_val { GNI_MSG_RENDEZVOUS_THRESHOLD,
 			   GNI_MR_SOFT_REG_LIMIT,
 			   GNI_MR_HARD_REG_LIMIT,
 			   GNI_MR_HARD_STALE_REG_LIMIT,
+			   GNI_IOV_LIMIT,
```

Comment:
No, as long as we have a hard upper limit set this should not be part the
domain ops.  My mistake.  I will remove it.

On Mon, Jun 27, 2016 at 12:34 PM, Sung-Eun Choi <notifications@github.com>
wrote:

> In prov/gni/include/fi_ext_gni.h
> <https://github.com/ofi-cray/libfabric-cray/pull/859#discussion_r68620449>
> :
>
> > @@ -62,6 +62,7 @@ typedef enum dom_ops_val { GNI_MSG_RENDEZVOUS_THRESHOLD,
> >  			   GNI_MR_SOFT_REG_LIMIT,
> >  			   GNI_MR_HARD_REG_LIMIT,
> >  			   GNI_MR_HARD_STALE_REG_LIMIT,
> > +			   GNI_IOV_LIMIT,
>
> GNI_IOV_LIMIT doesn't seem to result in any memory reduction or faster
> code paths. Am I missing something? If it doesn't affect performance in
> anyway, then it should be removed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ofi-cray/libfabric-cray/pull/859/files/dff688154f08afba43b1e034316579035ca21eea#r68620449>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHET8pymDXKwM9vU9nO2nKZSvmloxWquks5qQAnBgaJpZM4IzVqw>
> .
>